### PR TITLE
Remove trailing slash from /api/tiles call on frontend

### LINKED ISSUE #20090: Pins do not appear on pin map
**Describe the bug**
When you create a pin map using the data in the Sample Database, the pins do not appear.

**Logs**
Server logs contain a large number of:
```
[backend] 2022-01-31 19:23:20,110 WARN middleware.log :: GET /api/tiles/4/5/5/19/12/ 404 2.1 ms (0 DB calls)
[backend] "API endpoint does not exist."
```

**To Reproduce**
1. Go to Sample Database -> People -> Visualization
2. Click on "Map" to change the visualization type
3. Ensure that "Pin Map" is selected for map type, and that "Latitude" and "Longitude" are selected for the respective fields

**Expected behavior**
Pins should appear

**Information about your Metabase Installation:**
Current master

**Severity**
If this is a real bug, quite severe, since this visualization is broken

**Additional context**
I think this is a regression from https://github.com/metabase/metabase/pull/18912

### PR DESCRIPTION
**Pull request by noahmoss**
#18912 modified the path for this endpoint, and in the process removed a trailing slash (which makes it consistent with most of our other endpoints); however, the frontend call was not correspondingly updated to remove the slash as well. This broke pin maps since the calls now 404. This PR simply removers the slash from the sole frontend call site to restore pin map functionality.

It's a bit concerning that no tests caught this. Maybe this is a good case for a visual test but I don't know how to add those.

Fixes #20090

### CODE REVIEW

**Comment by rlotun** [comment]
or a simple api test to make sure you get a 200?

**Comment by dpsutton** [comment]
This one gets weird because this endpoint only gets hit if there are more than 1,000 tiles or locations. I remember recreating the issue manually and didn't notice any 400s or any missing tiles. But I think it can be difficult to find because there are already pins on the map and this api supplements those (i think. this stuff was really hard to follow)

**Review by dpsutton — APPROVED** [review]
Just recreated. So sorry about this.

**Comment by noahmoss** [comment]
@rlotun We do have API tests on the backend, but #18912 modified the endpoint used in the tests in conjunction with the actual code. I think only an e2e test would have caught this.
